src/components/ActivityList.tsx

```diff
@@ -0,0 +1,229 @@
+import { Button, Divider } from 'antd'
+import Loading from 'components/Loading'
+import { JuiceListbox } from 'components/inputs/JuiceListbox'
+import { PV_V1, PV_V2 } from 'constants/pv'
+import {
+  ProjectEventFilter,
+  ProjectEventsQueryArgs,
+  useProjectEvents,
+} from 'hooks/useProjectEvents'
+import { useMemo, useState } from 'react'
+import { AnyProjectEvent } from './activityEventElems/AnyProjectEvent'
+
+interface ActivityOption {
+  label: string
+  value: ProjectEventFilter
+}
+
+const ALL_OPT = (): ActivityOption => ({ label: t`All activity`, value: 'all' })
+
+const PV1_OPTS = (): ActivityOption[] => [
+  ALL_OPT(),
+  { label: t`Paid`, value: 'payEvent' },
+  { label: t`Redeemed`, value: 'redeemEvent' },
+  { label: t`Burned`, value: 'burnEvent' },
+  { label: t`Sent payouts`, value: 'tapEvent' },
+  { label: t`Sent reserved tokens`, value: 'printReservesEvent' },
+  { label: t`Edited cycle`, value: 'v1ConfigureEvent' },
+  { label: t`Deployed ERC20`, value: 'deployedERC20Event' },
+  { label: t`Transferred ETH to project`, value: 'addToBalanceEvent' },
+  { label: t`Created project`, value: 'projectCreateEvent' },
+]
+
+const PV2_OPTS = (): ActivityOption[] => [
+  ALL_OPT(),
+  { label: t`Paid`, value: 'payEvent' },
+  { label: t`Redeemed`, value: 'redeemEvent' },
+  { label: t`Burned`, value: 'burnEvent' },
+  { label: t`Sent payouts`, value: 'distributePayoutsEvent' },
+  {
+    label: t`Sent reserved tokens`,
+    value: 'distributeReservedTokensEvent',
+  },
+  { label: t`Edited cycle`, value: 'configureEvent' },
+  { label: t`Edited payout`, value: 'setFundAccessConstraintsEvent' },
+  { label: t`Transferred ETH to project`, value: 'addToBalanceEvent' },
+  { label: t`Deployed ERC20`, value: 'deployedERC20Event' },
+  {
+    label: t`Created a project payer address`,
+    value: 'deployETHERC20ProjectPayerEvent',
+  },
+  { label: t`Created project`, value: 'projectCreateEvent' },
+]
+
+const SHARED_OPTS = (): ActivityOption[] => [
+  ALL_OPT(),
+  { label: t`Paid`, value: 'payEvent' },
+  { label: t`Redeemed`, value: 'redeemEvent' },
+  { label: t`Burned`, value: 'burnEvent' },
+  { label: t`Sent payouts`, value: 'distributePayoutsEvent' },
+  { label: t`Sent payouts (v1)`, value: 'tapEvent' },
+  {
+    label: t`Sent reserved tokens`,
+    value: 'distributeReservedTokensEvent',
+  },
+  {
+    label: t`Sent reserved tokens (v1)`,
+    value: 'printReservesEvent',
+  },
+  { label: t`Edited cycle`, value: 'configureEvent' },
+  { label: t`Edited cycle (v1)`, value: 'v1ConfigureEvent' },
+  { label: t`Edited payout`, value: 'setFundAccessConstraintsEvent' },
+  { label: t`Transferred ETH to project`, value: 'addToBalanceEvent' },
+  { label: t`Deployed ERC20`, value: 'deployedERC20Event' },
+  {
+    label: t`Deployed project payer`,
+    value: 'deployETHERC20ProjectPayerEvent',
+  },
+  { label: t`Created project`, value: 'projectCreateEvent' },
+]
+
+export default function ActivityList({
```

Comment:
this component encapsulates all functionality required to show a filterable, infinite-paged list of projectEvents, including the option to provide a download modal component